build/gulpfile.vscode.js

```diff
@@ -500,7 +501,8 @@ BUILD_TARGETS.forEach(buildTarget => {
 
 		const vscodeTask = task.define(`vscode${dashed(platform)}${dashed(arch)}${dashed(minified)}`, task.series(
 			compileBuildTask,
-			compileExtensionsBuildTask,
+			cleanExtensionsBuildTask,
+			compileNonNativeExtensionsBuildTask,
```

Comment:
```suggestion
			compileExtensionsBuildTask,
```

?

Reply:
With this suggestion, we will be building the native ones twice. I separated them out so that we didn't double build based on your original feedback.

The non-native ones are built here, and then the native ones are part of `vscodeTaskCI` which is added to the end of this local build task.

Am I misunderstanding something?

Reply:
Sorry missed the tasks from `vscodeTaskCI`.